### PR TITLE
Add version tab which links to release notes to help tab

### PR DESCRIPTION
**Pull request by astrovsky01**
Based on https://github.com/galaxyproject/galaxy/issues/10937

### CODE REVIEW

**Comment by hexylena** [comment]
This looks good!

**Comment by dannon** [comment]
So it looks like admins would need to update the 'release url' every time they update, right?  Maybe instead of making this configurable, it's easier to just compose the correct version URL based on the internal version reckoning?

**Comment by hexylena** [comment]
> So it looks like admins would need to update the 'release url' every time they update, right?

I read it differently, that was just the base url for the documentation. It looked like the link was constructed on the fly to include _announce_user.html and the version string?

File: client/src/layout/menu.js [hunk 1]
```diff
@@ -192,6 +191,11 @@ export function fetchMenu(options = {}) {
                 title: _l("Interactive Tours"),
                 url: "tours",
             },
+            {
+                title: _l("Galaxy Version: " + Galaxy.config.version_major),
+                url: options.release_url + Galaxy.config.version_major + "_announce.html",
```
Comment (hexylena) [inline client/src/layout/menu.js]:
```suggestion
                url: options.release_url + Galaxy.config.version_major + "_announce_user.html",
```

maybe should be the user-facing release notes, more interesting for users.

**Comment by dannon** [comment]
@hexylena Yeah, it's composed -- I read through that too fast.  The naming of the option is a bit misleading, I'd have called it release_doc_base_url or something.

**Comment by hexylena** [comment]
Agreed! base would be better.

File: client/src/layout/menu.js (hunk 1 above)
Comment (astrovsky01) [reply client/src/layout/menu.js]: Good call, thanks!

**Comment by astrovsky01** [comment]
So the one thing I noticed when updating the name and just double checking everything was that, though we have a dev announcement page for 21.01, we don't for the user announcements. In the case where someone is running off of dev (for whatever reason), that link doesn't work, and won't until the official release. 
I guess the question is, should we create the same page stand-in for the user next release, or is there a better idea?

**Comment by dannon** [comment]
@astrovsky01 I'd actually go the other way with that and link just to the root release_version page.  And we should put at that root what most people expect to see.

So, instead of:

`https://docs.galaxyproject.org/en/master/releases/20.09_announce.html`

Simply: 
https://docs.galaxyproject.org/en/release_20.09/

(note `master` is going away anyway, to be `main` soon and this will need to be updated)

**Review by afgane — APPROVED** [review]
Tested this locally - lgtm.

**Pull request merged**